modules/Core/pages/profile.php

```diff
@@ -178,6 +190,18 @@
                                 ]
                             );
 
+                            EventHandler::executeEvent('userProfilePostReply', [
```

Comment:
Currently you use user language, you need to use default language

use `$default_language = new Language('core', DEFAULT_LANGUAGE);`